### ISSUE TITLE
Absurdly low-pitch audio (darth vader voice)

### ISSUE THREAD

**Issue by maltfield** [QubesOS/qubes-issues#6744]
**Qubes OS version**

```
Qubes release 4.0 (R4.0)
```

**Are you using any [testing](https://www.qubes-os.org/doc/testing/) repositories? If so, please list them AND where they are enabled (i.e., dom0, template, or standalone).**

No

**Are you providing feedback about a specific package or packages in testing? (If so, please list the package(s) here and [provide feedback in `updates-status`](https://www.qubes-os.org/doc/testing/#providing-feedback), linking to this issue.)**
No

**Affected component(s) or functionality**

Audio input & output

**Brief summary**

Sometimes when my laptop plays sounds (eg youtube) -- and, as of yesterday, when it records me through my microphone -- the sounds are extremely low-pitch. Everyone tells me on my video calls that they can barely understand me and that I sound like darth vader.

**How Reproducible**

 * (sometimes) play youtube videos and wonder why everyone's voice is suddenly possessed by daemons. Myley Cyrus is suddenly a stream of [death metal growls](https://en.wikipedia.org/wiki/Death_growl).

 * (always) connect to a video conference in slack or jitsi or whatever and get very disapproving/confused looks from coworkers who think I'm suddenly possessed by daemons

**To Reproduce**

1. Wake up
2. Grab a brush and put on a little makeup
3. Connect to weekly Monday standup video call
4. Experience the reactions of very confused/angry coworkers

**Expected behavior**

Sounds should play normally. My voice should be "recorded" normally.

**Actual behavior**

I sound like darth vader.

**Screenshots**

**Additional context**

Sorry, but I don't really know what more I can say to provide keywords and hopefully have other people chime-in who have this similar issue.

I'll say that I've been using QubesOS for years. Sometimes in the past years (also I believe on QubesOS R3.x), I would go to play a youtube video and the sound would be ridiculously & comically low-pitched. I found that it would just happen occasionally and that it would go away when I rebooted.

However, as of this week, I've had people tell me that they're experiencing the reverse: while my youtube videos play fine, my coworkers tell me that my voice is extremely low-pitched and they can't understand me. In best-cases it results in laughter. In worst cases, people aren't very happy that they can't communicate with me.

To help other people find this issue, I'm going to add this quote taken from wikipedia's article on "death growl"

> Death growls are also known as death metal vocals, brutal vocals, guttural vocals, death grunts, growled vocals, low pitched vocals, low growls, unclean vocals, harsh vocals, vocal fry, glottal fry, false cord vocals, death cord vocals and jocularly or disparagingly as "Cookie Monster vocals"

**Solutions you've tried**

My laptop is an asus ux331. Since Monday, I've tried:

1. Using my 3.5mm earphones w/ microphone. The issue persists.
2. Unplugging my earphones and using the built-in laptop microphone. The issue persists.
3. Trying a distinct VM. The issue persists.
4. Rebooting my computer. The issue persists.
5. Trying both slack and jitis. The issue is on both.
6. Trying to use my 3.5mm earphones and joining my video conference calls on my phone (instead of my laptop), and the issue goes away.

In conclusion, this appears to be a QubesOS-specific software bug (or at least something used by QubesOS)

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

I have no idea where to look for this. I searched the internet for "QubesOS" and "darth vader" and "audio" and "low-pitch"

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

I have no idea where to look for this. I searched the issues for "darth vader" and "audio" and "low-pitch"

**Comment by neowutran** [comment]
**I personally have no idea of the solution nor where to look**, except that you said that it stopped working Monday, do you mean that the day before the microphone worked correctly ? 
In that case, do you have an idea of what you updated in between ? ( template vm update / dom0 update / dom0 kernel update ? nothing ? )

**Comment by maltfield** [comment]
<object type="application/x-shockwave-flash" data="https://www.youtube.com/watch?v=rN38mC77DSM" width="640" height="360">
    <param name="movie" value="https://www.youtube.com/watch?v=rN38mC77DSM">
  </object>

 * https://www.youtube.com/watch?v=rN38mC77DSM

<iframe width="560" height="315" src="https://www.youtube.com/embed/rN38mC77DSM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
